controllers/storagecluster/cephfilesystem.go

```diff
@@ -27,6 +27,7 @@ func (r *StorageClusterReconciler) newCephFilesystemInstances(initStorageCluster
 		ObjectMeta: metav1.ObjectMeta{
 			Name:      generateNameForCephFilesystem(initStorageCluster),
 			Namespace: initStorageCluster.Namespace,
+			Labels:    cephv1.Labels{"rook_file_system": generateNameForCephFilesystem(initStorageCluster)},
```

Comment:
Why add this label to the filesystem CR? It's the same as the name of the CR, so it seems unnecessary.

Reply:
The podAffinity looks for this label of the pod, that why we need it

Reply:
The pod affinity looks on the pod labels, not the cephcluster CR labels